src/main/java/lotto/domain/lotto/Lotto.java

```diff
@@ -9,12 +10,29 @@
 
     private static final String NUM_LOTTO_ERROR_MESSAGE = "로또 번호는 %d개의 숫자로 이루어져야 합니다.";
     private static final String DUPLICATION_LOTTO_ERROR_MESSAGE = "로또 번호는 중복된 숫자가 존재할 수 없습니다.";
+    private static final String ERROR_COLLECT_NUMBER = "올바른 숫자를 입력하여 주세요";
+
+    private static final String REGEX = ", ";
+
     private final List<LottoNumber> numbers;
 
     private Lotto(List<LottoNumber> numbers) {
         this.numbers = numbers;
     }
 
+    public static Lotto generatedBy(String numbers) {
```

Comment:
메소드의 이름을 변경하여 보다 뚜렷하게 역할을 구분 지을 수 있을 것 같아요
```suggestion
    public static Lotto manual(String numbers) {
```

Reply:
변경했어요